internal/manager/config.go

```diff
@@ -284,6 +287,19 @@ func (c *Config) FlagSet() *pflag.FlagSet {
 	return flagSet
 }
 
+func (c *Config) Resolve() error {
+	if c.KongAdminToken == "" {
+		if c.KongAdminTokenPath != "" {
```

Comment:
Given `Validate` already is responsible for ensuring both are not set at the same time, I think it will be enough to just check `if c.KongAdminTokenPath != ""`.